phpBB/phpbb/db/migration/data/v330/storage_attachment_rename.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v330;
+
+class storage_attachment_rename extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
```

Comment:
Can you add an effectively installed checked for the config entry you're adding?

Reply:
Uhm, and what happen if there is a timeout or something doing the other part?